input/data/2008/2008ApJ...679.1427A/tev-000014.yaml

```diff
@@ -0,0 +1,35 @@
+# Source ID from TeGeV-cat: http://www.asdc.asi.it/tgevcat/)
+# LS I +61 303
+# VER J0240+612
+source_id: 14
+# Reference ID from ADS
+reference_id: 2008ApJ...679.1427A
+telescope: veritas
+
+##################
+# Source position fit
+pos:
+    # from fit to VERITAS data
+    ra: {val: 2h40m41s, err: 73s, err_sys: 90s }
+    dec: {val: +61d13m12s, err: 30s, err_sys: 90s }
+
+##################
+# Orbital period
+# (for binaries only)
+orbit:
+    # zero phase for orbital period from Gregory (2002)
+    mjd0: 43366.3     #
+    period: {val: 26.4960 }
+
+#############################################################
+# spectral fit results
+# (spectral fit for phases 0.5-0.8)
+
+spec:
+  type: pl
+  mjd: {min: 53999.4, max: 54147.1}
+  phase: {min: 0.5, max: 0.8 }
+  norm: {val: 2.89e-12, err: 0.32e-12, err_sys: 0.6e-12}
```

Comment:
Remove the `1e-12` part here when doing the data entry, i.e. change to this:
```
norm: {val: 2.89, err: 0.32, err_sys: 0.6}
```

We might change this in the future, but for now data entry has to be done this way.

The 1e-12 is noted as a scale with the unit in the spec:
https://github.com/gammapy/gamma-cat/blob/master/input/schemas/dataset_source_info.schema.yaml#L190